src/main.tsx

```diff
@@ -1,12 +1,9 @@
 import { StrictMode } from 'react';
 import { createRoot } from 'react-dom/client';
-import { RouterProvider } from 'react-router-dom';
-import CssBaseline from '@mui/material/CssBaseline';
-import router from './routes/routes.tsx';
+import App from './components/App.tsx';
 
 createRoot(document.getElementById('root')!).render(
   <StrictMode>
-    <CssBaseline />
-    <RouterProvider router={router} />
+    <App />
```

Comment:
@DamianStrz What do you think about moving the App file to the root folder, next to main.tsx?

Reply:
@Cubaa I think that's good idea.